src/tocbuilder.d

```diff
@@ -39,10 +53,16 @@ struct TocItem
 		}
 		if (hasChildren)
 		{
-			output.writeln(`<ul>`);
+            auto display = "";
+
+            if( module_parts.length >= parts.length )
+                if( parts == module_parts[ 0 .. parts.length ] )
```

Comment:
I'm not sure if I can read this correctly due to spacing but this could probably be written more readably as:

```
if(module_parts.startsWith(parts))
```

(using std.algorithm.startsWith)